components/Login.vue

```diff
@@ -99,6 +99,7 @@ const handleLogin = async () => {
       apiDomain.sitekey === "dev-nuxt-auth"
         ? "https://dev-nuxt-auth.a.kuroco.app"
         : `https://${apiDomain.sitekey}.g.kuroco.app`;
+    localStorage.setItem('sitekey', apiDomain.baseURL);
```

Comment:
should set `sitekey.value`, not `apiDomain.baseURL`.
Or, handling localStorage handler should be placed at one module I think.
How about moving those handlers into useAuth?